integration-test/data/FoD/fod-integation-test-data.json

```diff
@@ -10,8 +10,8 @@
 	"expectOutput": "default"
   },  
   {
-    "commands": "fod application create-web-app",
-    "arguments": "--owner=1851 --business-criticality=High --sdlc-status='Production' INT_T_App2:INT_T_App2_Release2",

```

Comment:
As above, if targetting  2.0.0 then `app-name:rel-name` is correct for creating a new release.